src/derivatives/atlas.md

```diff
@@ -493,10 +473,10 @@ A guide for using macros can be found at
    "ps13-pipeline": {
       "tpl-PS13": {
          "pet": {
-            "tpl-PS13_desc-nopvc_dseg.nii.gz": "",
-            "tpl-PS13_desc-pvc_dseg.nii.gz": "",
-            "tpl-PS13_dseg.json": "",
-            "tpl-PS13_dseg.tsv": "",
+            "atals-PS13_desc-nopvc_dseg.nii.gz": "",
```

Comment:
small typo in atlas. Is it clear from this that atlas belongs to the PS13 tpl and/or PS13 space?